PWGCF/TwoParticleCorrelations/TableProducer/identifiedBfFilter.h

```diff
@@ -59,6 +60,11 @@ enum MatchRecoGenSpecies {
   kWrongSpecies = -1
 };
 
+constexpr int pdgcodeEl = 11;
+constexpr int pdgcodePi = 211;
+constexpr int pdgcodeKa = 321;
+constexpr int pdgcodePr = 2212;
```

Comment:
Please consider using named constants from TPDGCode.h instead of numbers for the next iteration